### ISSUE TITLE
Dashboard : add extra filters on my proofs

### ISSUE THREAD

**Issue by raphodn** [openfoodfacts/open-prices-frontend#839]
### Story

Some users have many proofs, and want to easily filter through them (in order to find an existing proof for instance).

Currently there is only a filter to hide proofs with at least 1 price.

We could add new filters such as : 
- date (year, month, day)
- type (receipt or price tag or GDPR)
- location (brand, city, country ? slightly tricky)
- ?

**Comment by serpico** [comment]
Currently :

- clicking on the user chip send to the user page
- clicking on the date chip send the user to the date page
- clicking on the location chip send the user to the location page

However there's no effect ( not for me anyway ) when clicking on Price tag chip , is it the same for Receipt chip ?


![ChipToProof](https://github.com/user-attachments/assets/2493c352-0e32-4179-8d55-cffd9292d721)

**Comment by raphodn** [comment]
in the Proof modal, clicking on the "Price chip" should work if you're the price owner. And send you to a `/proofs/id` page.
though this is will probably change/be improved in the coming months, as there is more privacy-friendly features

**Comment by serpico** [comment]
@raphodn how much work would it take to add order like  :

- proof with the greater number of prices first/at the top vs oldest first/at the top
- oldest date first/at the top vs newest date first/at the top

although I could totally wait for the more versatile/powerful search  feature #839

**Comment by raphodn** [comment]
> proof with the greater number of prices first

isn't this already available ? or did you mean proof with least number of prices ?

**Comment by serpico** [comment]
> > proof with the greater number of prices first
> 
> isn't this already available ? or did you mean proof with least number of prices ? i feel this could be better managed with (super fine-grained) filters

Yeah it's available, but once you display the least number on top then you need the option greater number to switch back...

I agree with you about the super-fine grained filters...I mentioned it in my original message, on the PR but I failed to copy that part here.

**Comment by raphodn** [comment]
sorry i had removed your last sentence, i put it back ^^

and sorry but i still don't understand. there is already the option to order by "price_count DESC". you want to be able to order by "price_count ASC" as well ?

**Comment by raphodn** [comment]
> @snowflakes2499
We should allow sorting by proof date, missing prices, and total amount.
Possible fix: Modify the sorting function in this file to support different criteria using state.

In the dashboard it's already possible to order by proof date. you meant asc/desc ?
In the "all proofs" page the default order is by date desc.

|Current filters|Current order|
|-|-|
|![Image](https://github.com/user-attachments/assets/4a487a96-9f1d-4453-b3a9-3ef2f83e727c)|![Image](https://github.com/user-attachments/assets/153452dc-057e-4b7b-b508-18394658a28f)|

**Comment by raphodn** [comment]
Sorry for the late reply

Some quick info (not yet solving exactly the matter in hand)
- I just added a table view #958
- if you are on the desktop, you can change the url and reorder as you wish in the meantime
   - for instance after choosing the "addition date" order: https://prices.openfoodfacts.org/dashboard/proofs?order=-created
   - and you change the end param to `?order=created` (without the `-`)
